### PR TITLE
Feature/merge develop into main

### PR DESCRIPTION
**Pull request by DanOwenUKHO**
Merge develop into main

### CODE REVIEW

File: Deployment/src/Modules/APIM/variables.tf [hunk 1]
```diff
@@ -130,11 +140,14 @@ variable "pathsuffix" {
...
-  api_name              = local.env_name == "prod" ? "${var.api_name}${var.suffix}" : "${var.api_name} ${var.env_suffix[local.env_name]}${var.suffix}"
-  apim_api_path         = local.env_name == "prod" ? "${local.service_name}${var.pathsuffix}" : "${local.service_name}-${local.env_name}${var.pathsuffix}"
+  group_name            = local.env_name == "prod" ? "${var.group_name}${var.suffix}" : "${var.group_name} ${var.env_suffix[local.env_name]}"
+  product_name          = local.env_name == "prod" ? "${var.product_name}${var.suffix}" : "${var.product_name} ${var.env_suffix[local.env_name]}"
+  ui_product_name       = local.env_name == "prod" ? "${var.ui_product_name}${var.suffix}" : "${var.ui_product_name} ${var.env_suffix[local.env_name]}"
+  api_name              = local.env_name == "prod" ? "${var.api_name}${var.suffix}" : "${var.api_name} ${var.env_suffix[local.env_name]}"
+  ui_api_name           = local.env_name == "prod" ? "${var.ui_api_name}${var.suffix}" : "${var.ui_api_name} ${var.env_suffix[local.env_name]}"
+  apim_api_path         = local.env_name == "prod" ? "${local.service_name}${var.suffix}" : "${local.service_name}-${local.env_name}"
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply Deployment/src/Modules/APIM/variables.tf]: Seems the commit history is confused itself here/some merge has gone awry. I've reverted

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1246,7 +1259,7 @@ stages:
     dependsOn: 
       - vnextiatDeploy
     displayName: vNext E2E Deploy (inc terraform, webapp deploy)
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/develop'))
+    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/develop'), startsWith(variables['Build.SourceBranch'], 'refs/heads/dev/')))
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply azure-pipelines.yml]: Temporary change, revert back to only allow develop in e2e

File: global.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.423",
+    "version": "6.0.419",
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply global.json]: Revert change

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetByProductVersion.cs [hunk 4]
```diff
@@ -178,9 +178,12 @@ public async Task WhenICallTheApiWithAValidProductVersionWithCallbackURI_ThenASu
             CleanUpBatchIdList.Add(batchId);
         }
 
-        [TestCase("DE416080", 15, 1, TestName = "EditionNumber Unavailable")]
-        [TestCase("DE416080", 13, 2, TestName = "UpdateNumber Unavailable")]
-        [Category("SmokeTest-AIODisabled")]
+        //[TestCase("DE416080", 15, 1, TestName = "EditionNumber Unavailable")]
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetByProductVersion.cs]: Remove commented code

File: azure-pipelines.yml [hunk 5]
```diff
@@ -5,6 +5,10 @@ parameters:
     displayName: "Continue even if resources are getting destroyed"
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply azure-pipelines.yml:5]: I think this is correct, have remove the argument from the pipeline

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/appsettings.json [hunk 6]
```diff
@@ -68,12 +68,27 @@
...
-    "AioEditionNumber": 32,
-    "AioUpdateNumber": 14, ////The UpdateNumber here needs updating whenever QA data change happens
+    "AioEditionNumber": 31,
+    "AioUpdateNumber": 68, ////The UpdateNumber here needs updating whenever QA data change happens
     "EncCellName": "DE416080",
-    "EncEditionNumber": 13,
-    "EncUpdateNumber": 1, ////The UpdateNumber here needs updating whenever QA data change happens
-    "InvalidEncCellName": "US2ARCGE",
-    "InvalidAioCellName": "GZ800112"
+    //"EncEditionNumber": 13,
```
[the remark this answers is not in the document]
Comment (DanOwenUKHO) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/appsettings.json]: Removed commented lines

**Review by HugoBurgess — APPROVED** [review]
Looks good

**Pull request merged**